db/migrate/20230214205506_create_users.rb

```diff
@@ -2,7 +2,8 @@ class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :email
+      t.string :email, null: false
+      t.string :password_digest
```

Comment:
*Never, ever* edit an existing migration. Please see this snippet from the official Rails guide here: https://guides.rubyonrails.org/v7.0/active_record_migrations.html#changing-existing-migrations

Any time you need to modify an existing table, your best practice is to generate a generic migration that you manually edit to modify the table as you need.